webtau-groovy/src/test/groovy/com/twosigma/webtau/cfg/CLIArgsTest.groovy

```diff
@@ -0,0 +1,53 @@
+package com.twosigma.webtau.cfg
+
+import com.twosigma.webtau.utils.FileUtils
+import groovy.json.JsonOutput
+import org.junit.Test
+
+import java.nio.file.Paths
+import java.util.stream.Collectors
+
+class CLIArgsTest {
+    @Test
+    void "generate CLI docs"() {
+        try {
+            def handler = new WebTauGroovyCliArgsConfigHandler('--workingDir=${workingDir}', 'test.groovy')
+
+            WebTauConfig.registerConfigHandlerAsFirstHandler(handler)
+
+            def cfg = new WebTauConfig()
+
+            def cfgValues = cfg.getCfgValuesStream()
+            def cfgList = cfgValues.map { cfgValue ->
+                def defaultValue = cfgValue.key == 'workingDir' ? "" : (cfgValue.getAsString() ?: "")
```

Comment:
I would move this to a function to have a placeholder to add more exceptional cases